package.json

```diff
@@ -20,6 +20,7 @@
     "dependencies": {
         "@emotion/react": "^11.10.5",
         "@emotion/styled": "^11.10.5",
+        "@microsoft/eslint-formatter-sarif": "2.1.7",
```

Comment:
Why are we adding this package?

Reply:
i followed the error prompt after i made a PR,  i thought that's why i am getting the error

Reply:
i thought it will fix the error, maybe i should just remove them